src/data/storage.js

```diff
@@ -1,12 +1,41 @@
-const clearFns = []
+const customClearFns = []
 
 export function onClear(fn) {
-  clearFns.push(fn)
+  customClearFns.push(fn)
 }
 
-export function clear() {
-  while (clearFns.length) {
-    clearFns.pop()()
+export function runCustomClears() {
+  while (customClearFns.length) {
+    customClearFns.pop()()
   }
 }
 
+const SHORT = sessionStorage
+const LONG = localStorage
+export const LIFESPAN = Object.freeze({
+  SHORT,
+  LONG,
+})
+
+export function save(key, value, LIFESPAN) {
```

Comment:
I'd find it helpful to some how differentiate this `LIFESPAN` from the `LIFESPAN` above.

Reply:
agreed, https://github.com/silinternational/wecarry-ui/pull/129/commits/5e1327dfde1be65de225615bda547ca3a8d16ec4